main/src/library/Fixpoint/Interpreter.flix

```diff
@@ -16,27 +16,53 @@
 
 use Fixpoint/Ram.{RamStmt, RelOp, RamSym, RamTerm, BoolExp, RowVar, toDenotation};
 use Fixpoint/Ast.Denotation;
+use Fixpoint/Tuple.Tuple;
 
 namespace Fixpoint {
+    namespace Tuple {
```

Comment:
Why the extra namespace?

Reply:
I thought that Fixpoint.new/Fixpoint.toList would be ambigous and when Vector arrives, it will be easier to replace